doc/source/user_guide/categorical.rst

```diff
@@ -129,6 +129,48 @@ This conversion is likewise done column by column:
     df_cat['A']
     df_cat['B']
 
+Dummy / indicator / one-hot encoded variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some operations, like regression and classification,
+encodes a single categorical variable as a column for each category,
+with each row having False in all but one column (True).
+These are called dummy variables, or one-hot encoding.
+:class:`pandas.Categorical` objects can easily be converted to and from such an encoding:
+
+.. ipython:: python
+
+    cat = pd.Categorical(["a", "b", "b", "c"])
+    cat
+
+    dummies = cat.to_dummies()
+    dummies
+
+    pd.Categorical.from_dummies(dummies)
+
+The :meth:`pandas.Categorical.from_dummies` class method accepts a dataframe
+whose dtypes are coercible to boolean, and an ``ordered`` argument
+for whether the resulting ``Categorical`` should be considered ordered
```

Comment:
you need to have some good cross links to the current get_dummies section

otherwise this is very confusing

i would prefer that these are actually in the get_dummies with just a small note here